### ISSUE TITLE
Test suite failure

### ISSUE THREAD

**Issue by snoyberg** [louispan/data-distinct#3]
```
Failures:

  test/Data/Diverse/ManySpec.hs:33: 
  1) Data.Diverse.Many.Many is a Typeable
       expected: "Many (': * Int (': * Bool '[]))"
        but got: "Many (': * Int (': * Bool ('[] *)))"

  test/Data/Diverse/WhichSpec.hs:236: 
  2) Data.Diverse.Which.Which can be switched with a single 'CaseTypeable' handler
       expected: "Which (': * Int (': * Bool '[]))"
        but got: "Which (': * Int (': * Bool ('[] *)))"
```

**Comment by louispan** [comment]
@snoyberg The format of show TypeRep must have changed.
I won't have time to fix this right now, so you'll have to block data-diverse for the time being :(

**Comment by snoyberg** [comment]
Don't worry, I've only set the test suite to expected failure. Just send a
PR to remove the expected failure when you have a chance to fix it.

**Comment by louispan** [comment]
Fixed in d020ca2f5336c66688ac6af6ef011923dbc088e1
